databases/week2/part3Homework.sql

```diff
@@ -0,0 +1,23 @@
+-- Part 3: More queries
+
+-- Get all the tasks assigned to users whose email ends in @spotify.com 
+SELECT task.id,  task.title AS task_title FROM task
+LEFT JOIN user_task ON user_task.task_id=task.id
+WHERE user_task.user_id=2;
+ 
+-- Get all the tasks for 'Donald Duck' with status 'Not started'
+SELECT task.id,  task.title AS task_title, task.status_id AS task_status FROM task
+LEFT JOIN user_task ON user_task.task_id=task.id
+WHERE task.status_id=1
+AND user_task.user_id=11;
```

Comment:
Same as above. You don't necessarily know that there is only one user with that name.